packages/block-library/src/media-text/editor.scss

```diff
@@ -2,6 +2,7 @@
 	grid-template-areas:
 		"media-text-media media-text-content"
 		"resizer resizer";
+	align-items: center;
```

Comment:
Will adding this risk breaking existing Themes that have relied on there being no valign functionality? When we introduce this enhancement the goal is that nothing changes _until_ the user selects one of the v-align options.

This relates to the conversation regarding having a default setting applied on the topbar.

Reply:
This shouldn't change much as `align-items: center;` on the parent simply replaces the explicit `align-self: center;` on the child. Also this is just in the editor, and brings it in line with the styles that are being used on the Frontend in style.scss where `align-items: center;` on the parent is used.